pkg/node/manager/manager.go

```diff
@@ -385,6 +386,22 @@ func (m *manager) nodeIdentityLabels(n nodeTypes.Node) (nodeLabels labels.Labels
 	if m.conf.RemoteNodeIdentitiesEnabled() {
 		if n.IsLocal() {
 			nodeLabels = labels.NewFrom(labels.LabelHost)
+			if option.Config.EnableRemoteNodeIdentity {
```

Comment:
This option is wrong.
```suggestion
			if option.Config.PolicyCIDRSelectsNodes {
```